libqtile/backend/x11/core.py

```diff
@@ -171,6 +171,8 @@ def __init__(self, display_name: str | None = None) -> None:
 
         self.last_focused: base.Window | None = None
 
+        self.override_redirect_map: dict[int, window._Window] = {}
```

Comment:
May be badly named as rofi (as an example) does not set the override redirect window.